src/components/app/WebsocketConnectingOverlay.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Overlay, Spinner } from '@blueprintjs/core';
+
+import './WebsocketConnectingOverlay.css';
+
+function WebsocketConnectingOverlay({ isOpen }) {
+  return (
+    <Overlay
+      canEscapeKeyClose={false}
+      canOutsideClickClose={false}
+      isOpen={isOpen}>
+      <div id="websocketConnectingOverlay" className="pt-card pt-elevation-4">
+        <div>
+          <Spinner />
+        </div>
+        <div>Connecting to EtherDelta...</div>
```

Comment:
should this say ether or token delta?

Reply:
Yeap, this shows up when we're connecting to ED's API.